app/src/server.js

```diff
@@ -27,14 +28,16 @@ passport.use(
       "sessionStorage"
     ): SessionStorageInterface);
 
-    sessionStorage.get(token).then(
-      function(user) {
-        return done(null, user);
-      },
-      function() {
-        return done(null, false);
-      }
-    );
+    sessionStorage.get(token).then(function(maybeUser: Either<String, User>) {
```

Comment:
use anonymous function style consistent with the rest of the code `function(maybeUser: Either<String, User>)` ➡️ `(maybeUser: Either<String, User>) =>`